### ISSUE TITLE
The child config "authorization_server" under "league_oauth2_server" must be configured.

### ISSUE THREAD

**Issue by Elemor** [thephpleague/oauth2-server-bundle#64]
Hello!
I've tried to install this bundle via composer, but at the installation process, it caused this error:

The child config "authorization_server" under "league_oauth2_server" must be configured.

The "authorization_server" and "league_oauth2_server" are found only in vendor, exactly in league dir.

Any help will be great!

**Comment by Ak1r0** [comment]
Hi @Elemor make first the conf file as explain here https://github.com/thephpleague/oauth2-server-bundle/blob/master/docs/index.md and create the private and public keys, then you can run again the composer require cmd ;)

**Comment by johnvanham** [comment]
I have this issue as well on Symfony 5.4, seems like there must be a bug with the symfony flex recipe?

![image](https://user-images.githubusercontent.com/1741936/145555138-a52442a5-10ed-4943-96c7-19d7ca6aabc5.png)

**Comment by johnvanham** [comment]
I tried this again on Symfony 5.3 and it worked without needing to create the file manually. So that does suggest something isn't right with the recipe on Symfony 5.4?

**Comment by ash-m** [comment]
Same error on symfony 6.0.2
Note that installing trikoder/oauth2-bundle still works, but will spam your console with deprecation warnings.

**Comment by requtize** [comment]
Guys, first create YAML configuration, and then install bundle with composer. Doc describe installation process in failed order - to first install and then create config. I've already installed this bundle and all works perfectly.

@Ak1r0 Maybe the best solution will be when in the recipe You create a basic configuration for SF 5.x? This should fix this problem for the future.

**Comment by Myks92** [comment]
I met the same problem during installation. For security, the vendor assembly is required first, and then I copy the assembled vendor and configs to production. When using this bundle, the error is in the absence of a default configuration.

The solution is simple. Write a recipe for a bundle that, when installed, will copy the default config to the config folder.

Now in docker I just copy to `app/config/packages/league_oauth2_server.yaml` default config.

**Comment by chalasr** [comment]
Yep, we need a recipe. Thanks for the ping, I'll do that this week.

**Comment by Myks92** [comment]
> Yep, we need a recipe. Thanks for the ping, I'll do that this week.

Thank you. Waiting for recipe.

**Comment by jasonpeltzer** [comment]
> Yep, we need a recipe. Thanks for the ping, I'll do that this week.

Any news on the recipe?

**Comment by sdespont** [comment]
Thanks a lot for your work, but it seems like the recipe is still missing, @chalasr any news about it?

**Comment by chalasr** [comment]
Here is the recipe: https://github.com/symfony/recipes/pull/1130
Support reactions welcome there 🙏
